runtimes/peregrine/src/kilt/mod.rs

```diff
@@ -112,19 +114,19 @@ parameter_types! {
 }
 
 impl pallet_bonded_coins::Config for Runtime {
-	type BaseDeposit = runtime_common::constants::bonded_coins::BaseDeposit;
+	type BaseDeposit = ConstU128<{ BASE_DEPOSIT }>;
```

Comment:
I think it's good to keep namespacing constants with the module they belong to.

Reply:
made them explicit in: [5578ec5](https://github.com/KILTprotocol/kilt-node/pull/818/commits/5578ec5bc61d7e7b841915b0f37cd2f72dfc7231)

Reply:
Sorry to be picky, but we do import the `constants` module, so you can get rid of the `runtime_common` namespace.

Reply:
Changed it back in: [4296c3b](https://github.com/KILTprotocol/kilt-node/pull/818/commits/4296c3bf94f6a59af86e247192cafa43688f266f)